ncm-network/src/main/perl/network.pm

```diff
@@ -2112,7 +2148,13 @@ sub Configure
     #   1. stop everythig using old config
     #   2. replace updated/new config; remove REMOVE
     #   3. (re)start things
-    my $nwsrv = CAF::Service->new(['network'], log => $self);
+    my $service_name = "network";
+    if (!$self->_is_executable($CHKCONFIG_CMD) && defined($allow_nm) && $allow_nm) {
+        $service_name = "NetworkManager";
+    } else {
+        $self->error("chkconfig unavailable, NetworkManager not allowed");
```

Comment:
what does chkconfig have to do with NetworkManager?

Reply:
nothing, if chkconfig isn't found (as is the case on EL9), it will fallback to NetworkManager, if `allow_nm` is set to `true` in the profile.

Reply:
chkconfig can still be present because of other tools on the system that need it

Reply:
In that case I'll try adding a new schema entry to unambiguously choose the desired network service, and stop checking for the presence of `chkconfig`.

Reply:
> nothing, if chkconfig isn't found (as is the case on EL9), it will fallback to NetworkManager, if `allow_nm` is set to `true` in the profile.

chkconfig is found in rhel9, what fails is "chkconfig --level 2345 network on",  as there is no service Network on rhel9. on our rhel9 build we have to use chkconfig for other services (legacy), so this code will fail.